### PR TITLE
Template Fixes

### LINKED ISSUE #2: Network Name should be updated to 'casper'
https://github.com/nclouds/casper-infrastructure/blob/ab6108b64862dedf5068d307d29d4fd8251ab789/master.yml#L63

### LINKED ISSUE #3: NetworkIP should not be hardcoded / Ambiguous naming
We should encourage users to supply a known network IP here rather than hardcode one
eg. from the Peers list https://clarity.make.services/#/peers 

A more appropriate name for this var would be TrustedHashIP

https://github.com/nclouds/casper-infrastructure/blob/ab6108b64862dedf5068d307d29d4fd8251ab789/master.yml#L68W

### LINKED ISSUE #4: Replace deb curl commands with apt repository
Instructions can be found here: https://github.com/make-software/how-to-casperlabs/blob/master/docs/aws/setup-testnet-validator-from-scratch.md under **Install Casper node**

```
#Add Casper repository
#Execute the following in order to add the Casper repository to apt in Ubuntu.

echo "deb https://repo.casperlabs.io/releases" bionic main | sudo tee -a /etc/apt/sources.list.d/casper.list
curl -O https://repo.casperlabs.io/casper-repo-pubkey.asc
sudo apt-key add casper-repo-pubkey.asc
sudo apt update

#Install the Casper node software
sudo apt install casper-node-launcher -y
sudo apt install casper-client -y
```

https://github.com/nclouds/casper-infrastructure/blob/ab6108b64862dedf5068d307d29d4fd8251ab789/templates/casper-node.yml#L195

### LINKED ISSUE #5: Additional logs_collected for casper-node.stderr.log
https://github.com/nclouds/casper-infrastructure/blob/ab6108b64862dedf5068d307d29d4fd8251ab789/templates/casper-node.yml#L223

We also use an additional (non-json) log for stack traces/crashes: /var/log/casper/casper-node.stderr.log

### LINKED ISSUE #6: Importance of securely backing up the validator keypair should be highlighted in the README

### PR DESCRIPTION
**Pull request by ronaldour**
This update should address the following issues: #2 #3 #4 #5 #6

### CODE REVIEW

File: templates/casper-node.yml [hunk 1]
```diff
@@ -195,9 +195,9 @@ Resources:
                 curl -JLO https://bintray.com/casperlabs/debian/download_file?file_path=casper-client_0.9.3-0_amd64.deb
                 curl -JLO https://bintray.com/casperlabs/debian/download_file?file_path=casper-node-launcher_0.3.1-0_amd64.deb
                 apt install -y ./casper-client_0.9.3-0_amd64.deb ./casper-node-launcher_0.3.1-0_amd64.deb
-                sudo -u casper /etc/casper/pull_casper_node_version.sh 1_0_0 ${NetworkName}
+                sudo -u casper /etc/casper/pull_casper_node_version.sh 1_0_0 ${CasperNetwork}
                 sudo -u casper /etc/casper/config_from_example.sh 1_0_0
-                sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${NetworkIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml
+                sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${TrustedHashIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml
```
Comment (sacherjj) [inline templates/casper-node.yml]: Lets simplify:

sed -i "/trusted_hash =/c\trusted_hash = '$(curl -s ${TrustedHashIp}:8888/status | jq -r .last_added_block_info.hash | tr -d '\n')'" /etc/casper/1_0_0/config.toml

to 

sed -i "/trusted_hash =/c\trusted_hash = ${TrustedHash}'" /etc/casper/1_0_0/config.toml

File: templates/casper-node.yml [hunk 2]
```diff
@@ -45,13 +45,13 @@ Parameters:
...
+  CasperNetwork:
     Type: String
-    Description: Name of the network
+    Description: Name of the Casper network to join (e.g casper, casper-test)
 
-  NetworkIp:
+  TrustedHashIp:
     Type: String
-    Description: IP of the network
+    Description: IP used to obtain the trusted hash
```
Comment (sacherjj) [inline templates/casper-node.yml]: TrustedHashIp: -> TrustedHash:

Desc: trusted_hash from a valid node status endpoint or cspr.live site.

File: templates/casper-node.yml (hunk 1 above)
Comment (ronaldour) [reply templates/casper-node.yml]: Actually the file you commented is outdated. I changed the script to use this one https://github.com/make-software/how-to-casperlabs/blob/master/docs/aws/setup-testnet-validator-from-scratch.md#get-known-validator-ip Do you still want us to create the TrustedHash parameter?

**Comment by ronaldour** [comment]
3b2fa92 closes #1 ASG was replaced for an EC2 instance resource. The metrics and dashboard were updated also.

File: templates/casper-node.yml (hunk 1 above)
Comment (ronaldour) [reply templates/casper-node.yml]: Or should we remove it?

File: master.yml [hunk 3]
```diff
@@ -60,15 +60,14 @@ Parameters:
 
   #  ############## Service #############
 
-  NetworkName:
+  CasperNetwork:
     Type: String
-    Default: delta-11
-    Description: Name of the network
+    Default: casper
```
Comment (sacherjj) [inline master.yml]: I think this should default to casper-test rather than casper, as we want their first join to a TestNet rather than MainNet I think.

Comment (ronaldour) [reply master.yml]: Fixed in last commit

**Comment by rob-casper** [comment]
@ronaldour - Is this branch ready to be merged into master?

**Pull request merged**